db/migrate/20181220204317_add_original_url_to_redirection.rb

```diff
@@ -1,11 +1,5 @@
 class AddOriginalUrlToRedirection < ActiveRecord::Migration[5.2]
-  def up
+  def change
     add_column :redirections, :original_url, :text
-    update 'UPDATE redirections SET original_url = url'
```

Comment:
Hmm, why did this file need to change?

Reply:
Oof. Because I wasn't paying enough attention. Thank you.